server/routes/linkTokens.js

```diff
@@ -50,13 +55,25 @@ router.post(
         language: 'en',
         webhook: httpTunnel.public_url + '/services/webhook',
         access_token: accessToken,
+        transfer: {
+          intent_id: transferIntentId,
+        },
+        link_customization_name: 'transfer_ui_woo',
       };
 
       // If user has entered a redirect uri in the .env file
       if (redirect_uri.indexOf('http') === 0) {
         linkTokenParams.redirect_uri = redirect_uri;
       }
-      const createResponse = await plaid.linkTokenCreate(linkTokenParams);
+      const createResponse = await axios.post(
```

Comment:
any reason we can't continue using the plaid client here?

Reply:
does not support transferUI!!!